iceoryx_posh/test/integrationtests/test_roudi_roudi_environment.cpp

```diff
@@ -14,35 +14,24 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-#include "test.hpp"
 
-#include "iceoryx_posh/internal/roudi/roudi_lock.hpp"
+#include "iceoryx_posh/internal/roudi_environment/roudi_environment.hpp"
+
+#include "test.hpp"
 
 using namespace ::testing;
+using namespace iox::roudi;
 using ::testing::Return;
 
-
-class RouDiLock_test : public Test
+class RouDiEnvironment_test : public Test
 {
   public:
     void SetUp(){};
     void TearDown(){};
-
-    iox::roudi::RouDiLock sut;
 };
 
-TEST_F(RouDiLock_test, Lock)
-{
-}
-
-TEST_F(RouDiLock_test, TryDoubleLock)
-{
-    EXPECT_DEATH({ iox::roudi::RouDiLock sut2; }, ".*");
-}
-
-TEST_F(RouDiLock_test, LockAfterUnlock)
+TEST_F(RouDiEnvironment_test, StartingRouDiTwiceLeadsToError)
```

Comment:
I would say this isn't a RouDiEnvironment test but a test if the roudi lock works. You could achieve the same by just creating an instance of IceOryxRouDiComponents ... which would probably be better

Reply:
But it is carried out using the RouDiEnvironment object. It is now in integrationtests, so it does not have to follow unit test conventions where the Testfixture name and file name have to match the class name of the sut. So maybe this is a RouDiTest? Naming things again...
The test name itself is ok I guess.